CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Changelog
 
-## TBD
+## v0.1.0-alpha.0 (2023-06-26)
 
```

Comment:
Ideally we should callout the breaking change a little more – e.g. https://github.com/bugsnag/bugsnag-android-performance/blob/next/CHANGELOG.md?plain=1#L5-L6.

We don't (IMO) need a changelog for specifically the `alpha`, so if you can make sure that's added when we cut the full release that's fine.

Reply:
good point! I've backed out the version number change so this is now just a PR to bring the changelog up to date in advance of cutting the full release 🙂